### PR TITLE
Update jemalloc-pprof and tikv-jemallocator to latest version

### PR DESCRIPTION
**Pull request by umanwizard**

### CODE REVIEW

**Comment by umanwizard** [comment]
@antiguru please review the cargo-vet changes

File: misc/cargo-vet/audits.toml [hunk 1]
```diff
@@ -462,7 +462,7 @@ version = "3.0.4"
 [[audits.jemalloc_pprof]]
 who = "Moritz Hoffmann <mh@materialize.com>"
 criteria = "safe-to-deploy"
-version = "0.4.2"
```
Comment (antiguru) [inline misc/cargo-vet/audits.toml:465]: LGTM

File: misc/cargo-vet/audits.toml [hunk 2]
```diff
@@ -582,7 +582,7 @@ version = "0.11.3"
 [[audits.mappings]]
 who = "Moritz Hoffmann <mh@materialize.com>"
 criteria = "safe-to-deploy"
-version = "0.5.1"
```
Comment (antiguru) [inline misc/cargo-vet/audits.toml:585]: LGTM

File: misc/cargo-vet/audits.toml [hunk 3]
```diff
@@ -841,7 +841,7 @@ importable = false
 [[audits.pprof_util]]
 who = "Moritz Hoffmann <mh@materialize.com>"
 criteria = "safe-to-deploy"
-version = "0.4.0"
```
Comment (antiguru) [inline misc/cargo-vet/audits.toml:844]: LGTM

**Review by antiguru — APPROVED** [review]

**Review by def- — APPROVED** [review]

**Pull request merged**